crates/eframe/src/epi.rs

```diff
@@ -672,6 +674,28 @@ impl Frame {
         self.storage.as_deref()
     }
 
+    /// Request the current frame's pixel data. Needs to be retrieved by calling [`eframe::Frame::frame_pixels`]
+    /// during [`eframe::App::post_rendering`].
+    pub fn request_pixels(&mut self) {
+        self.output.pixels_requested = true;
+    }
+
+    /// Cancel a request made with [`eframe::Frame::request_pixels`].
+    pub fn cancel_request_pixels(&mut self) {
+        self.output.pixels_requested = false;
+    }
+
+    /// During [`eframe::App::post_rendering`], use this to retrieve the pixel data that was requested during
+    /// [`eframe::App::update`] via [`eframe::Frame::request_pixels`]. Currently only implemented with wgpu backend.
+    /// Returns None if
+    ///     Called in [`eframe::App::update`]
+    ///     [`eframe::Frame::request_pixels`] wasn't called on this frame during [`eframe::App::update`]
+    ///     The rendering backend doesn't support this feature (yet). Currently only implemented for the wgpu backend.
```

Comment:
Implementing it for `glow` shouldn't be much work at all. You can read from the framebuffer in OpenGL. See `examples/screenshot/src/main.rs` for an example!